software/spmd/striped_hello/main.c

```diff
@@ -127,10 +128,10 @@ int main()
 
     int bsg_id = bsg_x * bsg_tiles_X + bsg_y;
```

Comment:
BTW, shouldn't it be something like int bsg_id = bsg_Y * bsg_tiles_X + bsg_x;

i think bsg_Y, bsg_X are deprecated (replaced by __bsg_Y or something).
I would refer to bsg_set_tiles_x_y.h. I think it also calculate __bsg_id

Reply:
I'll change the test in another commit